.github/workflows/test.yml

```diff
@@ -55,3 +55,24 @@ jobs:
     - name: Test cibuildwheel
       run: |
         python ./bin/run_tests.py
+
+  test-emulated:
+    name: Test emulated cibuildwheel using qemu
+    runs-on: ubuntu-latest
+    timeout-minutes: 180
```

Comment:
This runs in parallel with the other runners, and takes less time (only 19m, because it's only  doing the emulation tests), so I don't think it costs us much